kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java

```diff
@@ -940,6 +962,32 @@ public boolean visible(String name, Map<String, Object> connectorConfigs) {
     }
   }
 
+  private static class S3StorageClassValidator implements ConfigDef.Validator {
+
+    public static final Map<String, StorageClass> TYPES_BY_NAME;
```

Comment:
I feel like this could be refactored to accept any `Enum`

Reply:
could not find a correct implementation in a static context